applications/serial_lte_modem/src/slm_at_host.h

```diff
@@ -53,20 +53,31 @@ void rsp_send(const uint8_t *str, size_t len);
 /**
  * @brief Request SLM AT host to enter data mode
  *
+ * No AT unsolicited message or command response allowed in data mode.
+ *
  * @param handler Data mode handler provided by requesting module
  *
- * @retval 0 If the operation was successful.
+ * @retval -EINPROGRESS If the operation was successful.
  *           Otherwise, a (negative) error code is returned.
```

Comment:
I prefer to keep 0 as successful return. -EINPROGRESS as successful returns conflicts the API style.

Reply:
OK, rollback